dotnetv3/Lambda/CreateDynamoDBTableExample/CreateDynamoDBTableExample/Function.cs

```diff
@@ -0,0 +1,103 @@
+using Amazon.DynamoDBv2;
+using Amazon.DynamoDBv2.Model;
+using Amazon.Lambda.Core;
+using Amazon.Lambda.RuntimeSupport;
+using Amazon.Lambda.Serialization.SystemTextJson;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+// This project specifies the serializer used to convert Lambda event into .NET classes in the project's main 
+// main function. This assembly register a serializer for use when the project is being debugged using the
+// AWS .NET Mock Lambda Test Tool.
+[assembly: LambdaSerializer(typeof(Amazon.Lambda.Serialization.SystemTextJson.DefaultLambdaJsonSerializer))]
+
+namespace CreateDynamoDBTableExample
+{
+    public class Function
+    {
+        /// <summary>
+        /// The main entry point for the custom runtime.
+        /// </summary>
+        private static async Task Main()
+        {
+            Func<string, ILambdaContext, Task> func = FunctionHandler;
+            using(var handlerWrapper = HandlerWrapper.GetHandlerWrapper(func, new DefaultLambdaJsonSerializer()))
+            using(var bootstrap = new LambdaBootstrap(handlerWrapper))
+            {
+                await bootstrap.RunAsync();
+            }
+        }
+
+        /// <summary>
+        /// Creates an Amazon DynamoDB table. The string input defines the name
+        /// of the new table.
+        /// </summary>
+        /// <param name="input">The name of the table to create.</param>
+        /// <param name="context">Optional object that contains information
+        /// about the function and the request. It is not used in this example.</param>
+        /// <returns></returns>
+        public static async Task FunctionHandler(string input, ILambdaContext context)
+        {
+            if (input is not null)
+            {
+                IAmazonDynamoDB client = new AmazonDynamoDBClient();
+                var tableName = input;
+
+                var response = await MakeTableAsync(client, tableName);
+
+                Console.WriteLine($"Created table: {response.TableDescription.TableName}.");
+            }
+        }
+
+        /// <summary>
```

Comment:
You haven't told me what the two attributes mean and what they imply.
"The created table has two string attributes: ID and Area. The ID ... and the Area ..."

Reply:
Added descriptions for the partition key and the sort key.